modules/configuring-a-pipeline-server.adoc

```diff
@@ -44,6 +44,7 @@ ifdef::upstream[]
 endif::[]
 * You have created a data science project that you can add a pipeline server to.
 * You have an existing S3-compatible object storage bucket and you have configured write access to your S3 bucket on your storage account.
+* If you are configuring a pipeline server for production pipeline workloads, you have an existing external MySQL or MariaDB database.
```

Comment:
@chtyler is there a better phrase I can use for "production pipeline workloads"?

Reply:
Fair question. The term "production pipeline workloads" does sound a little wordy to me, but it does add clarity at the same time, as it provides a differentiation between the pipelines that are in production and those that aren't. I'd be tempted to leave this as it is.